### ISSUE TITLE
Clarification on Version 0.10.0 and 0.10.1 Fork

### ISSUE THREAD

**Issue by ppcfan** [argmaxinc/WhisperKit#287]
Could you please clarify why versions 0.10.0 and 0.10.1 have diverged? This is rather confusing. It seems that version 0.10.0 not only updated the interface but also fixed some bugs. However, these fixes do not appear to have been merged into version 0.10.1. Could you share the next steps in your plan? Thank you!

**Comment by atiorh** [comment]
Hi @ppcfan, @ZachNagengast will chime in soon but here is a quick summary: `v0.10.0` will be yanked due to an issue so please rely on `v0.10.1`.

**Comment by ZachNagengast** [comment]
Yep, sorry for the confusion @ppcfan the release branch was created based on the pre-squashed version of this pull request #282, which then became the branch that went into the release and had different histories. The diffs are actually the same if you compare the diff, just has different commits. Will remove that version since the patch v0.10.1 has the history from main.

Diffs for comparison:
https://github.com/argmaxinc/WhisperKit/compare/v0.10.1...2b4c011b76bce78b4202f756cd2a9b94d582c850%5E%5E%5E
